redis/cache/redisx/redisxtest/doc.go

```diff
@@ -0,0 +1,4 @@
+// Package redisxtest provides utilities for baseplate.go/redids/cache/redisx testing.
```

Comment:
no need to spell out the full `baseplate.go/redids/cache/redisx`, just say "... for redisx testing" is enough.